source/PlayerInfo.cpp

```diff
@@ -2340,6 +2340,8 @@ void PlayerInfo::StepMissions(UI *ui)
 				if(ship->IsDestroyed())
 					mission.Do(ShipEvent(nullptr, ship, ShipEvent::DESTROY), *this, ui);
 	
+	string visitText;
+	int visitDialogs = 0;
```

Comment:
The first visitDialog is not counted in the code below. Maybe call the variable `extraVisitDialogs` or `otherVisitDialogs`.